common/src/main/java/org/apache/nemo/common/ir/IRDAG.java

```diff
@@ -710,7 +710,7 @@ public IREdge getEdgeById(final String id) {
 
   @Override
   public IREdge getEdgeBetween(final String srcVertexId,
-                               final String dstVertexId) throws IllegalEdgeOperationException {
```

Comment:
As far as I can check, this change has to be made on both the `common/src/main/java/org/apache/nemo/common/dag/DAG.java` and the `common/src/main/java/org/apache/nemo/common/dag/DAGInterface.java` files, as mentioned [here](https://sonarcloud.io/project/issues?fileUuids=AWkltI3qgtFwgfOLu2ET%2CAWkltI3qgtFwgfOLu2EX&id=org.apache.nemo%3Anemo-project&resolved=false&rules=squid%3ARedundantThrowsDeclarationCheck) Could you check and confirm and make the appropriate changes there, if applicable? Thanks! 😄